helios-testing/src/main/java/com/spotify/helios/testing/HeliosSoloDeployment.java

```diff
@@ -349,9 +319,7 @@ private String getHostPort(final String containerId, final int containerPort)
       final NetworkSettings settings = dockerClient.inspectContainer(containerId).networkSettings();
       for (Map.Entry<String, List<PortBinding>> entry : settings.ports().entrySet()) {
         if (entry.getKey().equals(heliosPort)) {
-          for (PortBinding portBinding : entry.getValue()) {
-            return portBinding.hostPort();
-          }
+          return entry.getValue().get(0).hostPort();
```

Comment:
Out of curiosity, is this purely stylistic or are there other advantages in Java? For whatever reason I generally use the loop, but I don't feel strongly about it.